simpeg_drivers/utils/tile_estimate.py

```diff
@@ -122,6 +122,9 @@ def run(self) -> SimPEGGroup:
         """
         Run the tile estimator.
         """
+        # TODO find out why this is needed. Without I get an error because the
+        # data_object is no longer the parent of tmi_channel.
+        _ = self.driver.inversion  # Triggers creation of something
```

Comment:
Probably because the data class copies the survey without data, and re-assign the data after.